internal/engine/crd.go

```diff
@@ -267,12 +269,14 @@ func convertCRD(crd cue.Value) (*IntermediateCRD, error) {
 							Label: ast.NewIdent("spec"),
 							Value: ast.NewIdent("#" + kname + "Spec"),
 						}
+						specref.Constraint = token.NOT
 						astutil.CopyComments(specref, x)
 						cursor.Replace(specref)
 						return false
 					case "status":
-						statusf = new(ast.Field)
-						*statusf = *x
+						//TODO: decide if status should be included
```

Comment:
yeah this felt wonky while i was working on it

Reply:
Yeah I decided to remove status for now, Timoni strips the status at apply-time same as kubectl, so having it there just inflates the files with no real benefit.